### PR TITLE
Add queries on SmartScreen blocks

### PR DESCRIPTION
**Pull request by TomerAlpert**

### CODE REVIEW

File: Hunting Queries/Protection events/SmartScreen URL block ignored by user.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Query for SmartScreen URL blocks, where the user has decided to run the malware nontheless.
+// An additional optional filter is applied to query only for cases where Edge has downloaded a file shortly after the ignored block.
```
Comment (meni-braun) [inline Hunting Queries/Protection events/SmartScreen URL block ignored by user.txt]: Microsoft Edge?

File: Hunting Queries/Protection events/SmartScreen app block ignored by user.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            // Filter out SmartScreen test files downloaded from https://demo.smartscreen.msft.net/
+            and not (FileName startswith "knownmalicious" and FileName endswith ".exe")
+    | project EventTime, ComputerName, BlockedFileName=FileName, SHA1, Experience=tostring(parse_json(AdditionalFields).Experience), ActivityId=tostring(parse_json(AdditionalFields).ActivityId), InitiatingProcessFileName;
+// Query for UserDecision events - each one means the user has decided to ignore the warning and run the app.
+let userIgnoredWarning=
+    MiscEvents
+    | where ActionType == "UserDecision"
+    | project ComputerName, ActivityId=tostring(parse_json(AdditionalFields).ActivityId);
+// Join the block and user decision event using an ActivityId
+let ignoredBlocks = smartscreenAppBlocks | join kind=leftsemi (userIgnoredWarning) on ComputerName, ActivityId | project-away ActivityId;
```
Comment (meni-braun) [inline Hunting Queries/Protection events/SmartScreen app block ignored by user.txt]: Please break each | to a separate line

File: Hunting Queries/Protection events/SmartScreen app block ignored by user.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// Query for SmartScreen application blocks on files with "Malicious" reputation, where the user has decided to run the malware nontheless.
+// Read more about SmartScreen here: https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview
+// Data availability: These events are available only on Windows 10 version 1703 and onwards.
+// Tags: #SmartScreen
+let smartscreenAppBlocks = 
+    MiscEvents
```
Comment (meni-braun) [inline Hunting Queries/Protection events/SmartScreen app block ignored by user.txt]: No time filter in this query.
Best practices are to include filter on time, to filter as much data as possible prior to additional operations.

File: Hunting Queries/Protection events/SmartScreen URL block ignored by user.txt (hunk 1 above)
Comment (TomerAlpert) [reply Hunting Queries/Protection events/SmartScreen URL block ignored by user.txt]: Resolved

**Pull request merged**